README.md

```diff
@@ -1,54 +1,272 @@
+        * [Get recordings](#get-recordings)
+        * [Publish recordings](#publish-recordings)
+        * [Delete recordings](#delete-recordings)
+* [Submitting bugs and feature requests](#submitting-bugs-and-feature-requests)
 
-[![@bigbluebutton on Twitter](https://img.shields.io/badge/twitter-%40bigbluebutton-blue.svg?style=flat)](https://twitter.com/bigbluebutton)
-![Website](https://img.shields.io/website-up-down-green-red/http/bigbluebutton.org.svg?label=BigBlueButton.org)
 
-[![PHP 5.5](https://img.shields.io/badge/php-5.5-f33.svg?style=flat-square)](https://php.net/)
-[![PHP 5.6](https://img.shields.io/badge/php-5.6-f33.svg?style=flat-square)](https://php.net/)
-[![PHP 7.0](https://img.shields.io/badge/php-7.0-f33.svg?style=flat-square)](https://php.net/)
-[![PHP 7.1](https://img.shields.io/badge/php-7.1-f93.svg?style=flat-square)](https://php.net/)
-[![PHP 7.2](https://img.shields.io/badge/php-7.2-9c9.svg?style=flat-square)](https://php.net/)
-[![PHP 7.3](https://img.shields.io/badge/php-7.3-9c9.svg?style=flat-square)](https://php.net/)
+## :question: Why should I use a fork?
+To explain why you should use a fork, we have to explain why we created our own
+fork. While BigBlueButton is a great product, contributing as an external
+developer is often cumbersome. Bug fixes and features are not merged. Modern
+enhancements are postponed or not allowed. Therefore 4 people from different
+companies and universities decided to create a open minded alternative to the
+existing PHP API. While we are still backwards compatible, this fork has the
+following advantages:
 
-## Installation and usage
+- Don't require unsecure environment variables
+- Tests are split into unit and integration tests and therefore working
+- Development is simplified through git hooks and contributor guidelines
+- Documentation is up-to-date and complete
+- API is fixed and extended to exploit the full potential
 
-The [wiki] contains all the documentation related to the PHP library. We have also written a samples to show a full
-install and usage example.
+## :gear: Installation and usage
+### Requirements
+In order to use this library you have to make sure to meet the following
+requirements:
 
-## Submitting bugs and feature requests
+- PHP 5.4 or above.
+- curl library installed.
+- mbstring library installed.
+- xml library installed.
+
+### Installation
+We are using composer to manage and install all libraries, so you need to use it
+too. If you setup composer for your project, you just have to add this API to
+your required section via
+
+```
+composer require littleredbutton/bigbluebutton-api-php:dev-master
+```
+
+### Basic usage
+To make any requests to your BigBlueButton instance, you have to create an API object:
+
+```php
+use BigBlueButton/BigBlueButton;
+
+$bbb = new BigBlueButton($apiUrl, $apiSecret);
+```
+
+If you didn't use composer before, make sure that you include `vendor/autoload.php`.
+
+#### Test if API url and secret are valid
```

Comment:
That is ok for now, but a new function for connectivity check would be great. Should I create a new issue?

Reply:
A pr would be better 😉

Reply:
i think we need to add api documentation as well for each function so developer can easily find his needs(understand)  and add [apimate](https://mconf.github.io/api-mate/) url for valid  bigbluebuttonserver api calls.

Reply:
There are plenty of ways to improve the documentation, but can we merge this as a first step?

Reply:
I'm agree with @sualko. The first step can be merge the PR, and then, we can add more improvements.

Reply:
@sualko i just suggest to all but  first merge this pr on next time we can improve the documentation.